src/Ocelot/WebSockets/Middleware/WebSocketsProxyMiddleware.cs

```diff
@@ -119,11 +127,10 @@ private static async Task Proxy(HttpContext context, string serverEndpoint)
 
             var destinationUri = new Uri(serverEndpoint);
             await client.ConnectAsync(destinationUri, context.RequestAborted);
-            using (var server = await context.WebSockets.AcceptWebSocketAsync(client.SubProtocol))
-            {
-                var bufferSize = DefaultWebSocketBufferSize;
-                await Task.WhenAll(PumpWebSocket(client, server, bufferSize, context.RequestAborted), PumpWebSocket(server, client, bufferSize, context.RequestAborted));
-            }
+
+            using var server = await context.WebSockets.AcceptWebSocketAsync(client.SubProtocol);
+            var bufferSize = DefaultWebSocketBufferSize;
```

Comment:
bufferSize is useless

Reply:
✔️ Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1377/commits/e48e04aaa63f7ab00bb7d3356b3687a3f8ceb14c